addon/components/drag-sort-item.js

```diff
@@ -65,10 +67,9 @@ export default Component.extend({
   isDragged2     : false,
   originalHeight : null,
 
-  shouldShowPlaceholderAbove2 : undefined,
-  shouldShowPlaceholderBelow2 : undefined,
-
-
+  shouldShowPlaceholderBefore2 : undefined,
+  shouldShowPlaceholderAfter2  : undefined,
+  isVertical                   : not('dragSort.isHorizontal'),
 
   // ----- Aliases -----
   isDraggingUp : reads('dragSort.isDraggingUp'),
```

Comment:
This name is now improper. But I can't think of a better name. Let's keep it as it is.